README.md

```diff
@@ -56,12 +65,12 @@ The following commands install and run the development version of The Lounge:
 ```sh
 git clone https://github.com/thelounge/lounge.git
 cd lounge
-npm install
-NODE_ENV=production npm run build
-npm start
+yarn install
```

Comment:
Just `yarn` is enough

Reply:
We know. Opted to keep this way because it's not immediately obvious for people that haven't used yarn.